File: Assets/Plugins/Broadcast.js
#pragma strict

static function SendMessage (messageName: String) {
    for (var go : GameObject in GameObject.FindObjectsOfType(GameObject)) {
        go.SendMessage(messageName, SendMessageOptions.DontRequireReceiver);
    }
}

static function SendMessage (messageName: String, messageParameter: Object) {
    for (var go : GameObject in GameObject.FindObjectsOfType(GameObject)) {
        go.SendMessage(messageName, messageParameter, SendMessageOptions.DontRequireReceiver);
    }
}

File: Assets/Scripts/Backup/Broadcast.cs
using UnityEngine;

public static class Broadcast
{
    public static void SendMessage(string messageName)
    {
        foreach (GameObject go in Object.FindObjectsOfType<GameObject>())
        {
            go.SendMessage(messageName, SendMessageOptions.DontRequireReceiver);
        }
    }

    public static void SendMessage(string messageName, object parameter)
    {
        foreach (GameObject go in Object.FindObjectsOfType<GameObject>())
        {
            go.SendMessage(messageName, parameter, SendMessageOptions.DontRequireReceiver);
        }
    }
}
